python-threatexchange/threatexchange/TE.py

```diff
@@ -181,8 +181,8 @@ def getInfoForIDs(self, ids, **kwargs):
     # Gets threat updates for the given privacy group.
     @classmethod
     def getThreatUpdates(self, privacy_group, **kwargs):
-        if "next" in kwargs:
-            url = kwargs["next"]
+        if kwargs["next_page"] is not None:
```

Comment:
The way experimentalfetch is written next_page is always passed in with None meaning start the query using the other given params (i.e. start time, stop time, threat types) and a non-none value meaning fetch the next page, so it was slightly cleaner to not check next_page in kwargs.

Can update if that style is preferred

Reply:
Change Request:

Since it is an expected parameter and not one that we are allowing for dynamic support in the URL building below (like the other keyword arguments), can you please switch `next_page` to be a optional parameter instead of a kwarg.

Something like 

```python
def getThreatUpdates(self, privacy_group, next_page = None, **kwargs):
```

Reply:
Updated